docs-src/src/ExampleSection.js

```diff
@@ -1,22 +1,65 @@
-import React from 'react'
+import React, { useEffect } from 'react';
```

Comment:
seems like theres some formatting inconsistencies with spaces between curlies and if statement parenths, but nbd.